starter-code/bin/www

```diff
@@ -25,7 +25,7 @@ var server = http.createServer(app);
  * Listen on provided port, on all network interfaces.
  */
 
-server.listen(port);
+server.listen(port,  () => console.log (`listen on ${port}`));
```

Comment:
🥇 GREAT JOB this console.log! I really appreciate that you played with the code (especially one that you didn't write) to have a clearer view of what was happening!